docs/kube-burner.md

```diff
@@ -36,6 +36,15 @@ Each iteration of this workload creates the following objects:
   - 1 deployment holding a client application for the previous database
   - 1 service pointing to the postgresl database
 
+- **max-namespaces**: This workload is a cluster limits focused test which creates maximum possible namespaces across the cluster. This is a namespaced workload, meaning that kube-burner **will create as many namespaces with these objects as the configured job_iterations**.
+  - 1 deployment holding a postgresql database
+  - 5 deployments consisting of a client application for the previous database
+  - 1 service pointing to the postgresl database
+
+- **max-services**: This workload is a cluster limits focused test which creates maximum possible services per namespace. It **will create a single namespace, each iteration of this workload will populate that namespace with these objects:**
```

Comment:
The number of pods/deployments that can run with 5000 services in the namespace is limited by the byte size of the service name as there's a limit on the size of the arguments that be passed during exec. Should we expose the service name as a config option to be able to tweak it during the test? Thoughts?

Reply:
As we discussed offline, I removed UUID suffix from resource names. so we shouldn't overcome the byte size limit.